### ISSUE TITLE
Cannot reproduce the results in Table 4 of the paper

### ISSUE THREAD

**Issue by thudzj** [lolemacs/soft-sharing#2]
Hi Savarese,

Thanks a lot for sharing the codes. I have run the SWRN-28-10-6 model with cutout regularization and all the 50000 training images, yet I could not obtain the 2.7% test error reported in the paper. I am wondering if I still need more modifications to achieve that. I appreciate it if you can give some useful suggestions.

Thanks!
Zhijie

**Comment by Johswald** [comment]
Hey Pedro,

I also can't reproduce your results. For CIFAR10 or CIFAR100, with/out cutoff augmentation. 
Please clarify how you to use the code. Thanks!

**Comment by Johswald** [comment]
Just for anybody who is trying to reproduce the results, leaving the validation set size to 0 and considering the final test acc might help.
